contracts/RiskManagerV1.sol

```diff
@@ -145,6 +157,11 @@ contract RiskManagerV1 is Auctioneer, Ownable {
     /// @notice Creates an auction for tbtc deposit in liquidation state.
     /// @param  depositAddress tBTC Deposit address
     function notifyLiquidation(address depositAddress) external {
+        require(
+            tbtcDepositToken.exists(uint256(depositAddress)),
+            "Address is not a deposit contract"
+        );
+
```

Comment:
Now the `notifyLiquidated` will revert if called for a deposit without prior `notifyLiquidation` call (`depositToAuction` will return zero as auction address). However, I agree it will be better to make an explicit check here for the future.

Reply:
Done in https://github.com/keep-network/coverage-pools/pull/77/commits/dd5dcc8fa345dbd73d46dd9cf2cb2ac97d03f7a6